src/Sparrow/Json/BlittableJsonTextWriter.cs

```diff
@@ -173,13 +172,59 @@ public unsafe void WriteString(LazyStringValue str)
 
                 EnsureBuffer(1);
                 _buffer[_pos++] = Quote;
+
                 return;
             }
 
             UnlikelyWriteEscapeSequences(str, numberOfEscapeSequences, escapeSequencePos, strBuffer, size);
+
+            EnsureBuffer(1);
+            _buffer[_pos++] = Quote;
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public void WriteStringJoin(LazyStringValue[] strArray)
```

Comment:
No, this method needs to be deleted. You need to use a json array instead. We don't do string parsing.